nevergrad/benchmark/experiments.py

```diff
@@ -960,26 +960,26 @@ def rocket(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                             yield xp
 
 
-@registry.register
-def mixsimulator(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
-    """MixSimulator of power plants
-    Budget 20, 40, ..., 1600.
-    Sequential or 30 workers."""
-    funcs = [OptimizeMix()]
-    seedg = create_seed_generator(seed)
-    optims: tp.List[str] = get_optimizers("basics", seed=next(seedg))  # type: ignore
-
-
-    seq = np.arange(0, 1601, 20)
-    for budget in seq:
-        for num_workers in [1, 30]:
-            if num_workers < budget:
-                for algo in optims:
-                    for fu in funcs:
+try:  # Dirty try/except because MixSimulator is not up to date.
+    OptimizeMix()
+    @registry.register
+    def mixsimulator(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+        """MixSimulator of power plants
+        Budget 20, 40, ..., 1600.
+        Sequential or 30 workers."""
+        fu = OptimizeMix()
+        seedg = create_seed_generator(seed)
+        optims: tp.List[str] = get_optimizers("basics", seed=next(seedg))  # type: ignore
+        seq = np.arange(0, 1601, 20)
+        for budget in seq:
+            for num_workers in [1, 30]:
+                if num_workers < budget:
+                    for algo in optims:
                         xp = Experiment(fu, algo, budget, num_workers=num_workers, seed=next(seedg))
                         if not xp.is_incoherent:
                             yield xp
-
+except AttributeError as e:
+    assert "optimizerlib" in str(e), e
```

Comment:
what is the message? a small comment would be helpful, right now I don't understand what optimizerlib has to do with it.